able.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * in-memory cache
+ */
+
+var enabled = [];
+var disabled = [];
+
+/**
+ * Checks if the specified `ns` is enabled.
+ *
+ * @param {String} ns  Wildcards are not supported here
```

Comment:
It's been a really long time... but I think it was just for simplicity. Imagine I've enabled `mocha*` but disabled `mocha:runner`, what would you expect `enabled("mocha*")` to return? (since there is a clear exception to this rule that makes the wildcard _technically_ can't be true 100% of the time)